stock_scanner_inventory/i18n/stock_scanner_inventory.pot

```diff
@@ -1,81 +0,0 @@
-# Translation of Odoo Server.
```

Comment:
Don't remove the POT file. Due to the new translation platform, is now needed. It's going to be auto-generated, but removing it, this makes the diff larger.